.github/actions/build_deb.sh

```diff
@@ -17,9 +33,19 @@ cd libsilkit-${SILKIT_VERSION}/
 
 echo "Running dh_make"
 dh_make -ly
+ret_val=$?
+if [ "$ret_val" -gt '1' ] ; then
```

Comment:
style, using `if test` everywhere, using the alias `if [` here.

Reply:
Done, changed all to the `if [` style